src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/help/Add-AzureAnalysisServicesAccount.md

```diff
@@ -62,6 +113,34 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ServicePrincipal
+{{Fill ServicePrincipal Description}}
```

Comment:
@liangyong79 please fill out this description if you decide to keep this parameter